src/main/java/org/asteriskjava/manager/event/QueueMemberPauseEvent.java

```diff
@@ -118,21 +117,6 @@ public void setRinginuse(Boolean ringinuse)
         this.ringinuse = ringinuse;
     }
 
-    /**
-     * @return the iface
-     */
-    public String getInterface()
```

Comment:
I gatther that the interface is no longer available in this event.
Should we be making it available for backwards compatibility?

Reply:
Yes, if you use asterisk prior to 12 asterisk will deliver location as field, interface is unknown and vice versa